src/opus_encoder.c

```diff
@@ -1668,7 +1675,6 @@ opus_int32 opus_encode_native(OpusEncoder *st, const opus_val16 *pcm, int frame_
     if (st->mode != MODE_CELT_ONLY)
     {
         opus_int32 total_bitRate, celt_rate;
-        opus_int activity;
```

Comment:
could drop `activity = VAD_NO_DECISION` below now

Reply:
Done. Thanks for pointing that out.